src/lib.rs

```diff
@@ -85,27 +86,27 @@ impl<F: Field, PC: PolynomialCommitment<F>> SemanticallyValid for BatchLCProof<F
 /// a sender to commit to multiple polynomials and later provide a succinct proof
 /// of evaluation for the corresponding commitments at a query set `Q`, while
 /// enforcing per-polynomial degree bounds.
-pub trait PolynomialCommitment<F: Field>: Sized {
+pub trait PolynomialCommitment<G: AffineCurve>: Sized {
```

Comment:
/// Describes the interface for a linear homomorphic polynomial commitment scheme.
/// Such a scheme is a commitment scheme for polynomials which is linear, i.e.
///         a*Comm(p(X);r) + b*Comm(q(X);s) = Comm(a*p(X)+b*q(X);r+s),
/// where r,s are the commitment randomnesses, and which additionally comes 
/// with a succinct (non-interactive) evaluation proof.
/// The simple single-polynomial evaluation proof depends on the conrete scheme, 
/// whereas more expressive opening proofs are reduced to a simple one by leveraging
/// linearity.

Reply:
Done